### PR TITLE
Monitor output wrong

### PR DESCRIPTION
**Pull request by ineiti**
Even with a nice test.

### CODE REVIEW

**Comment by Liamsi** [comment]
Looks good. You did run the simulations again, I guess? If so, I'll quickly resole the merge conflicts and merge.

File: lib/monitor/stats.go [hunk 1]
```diff
@@ -337,129 +352,58 @@ func (t *value) Collect(measure string, df DataFilter) {
...
+func (t *Value) HeaderFields() []string {
+	return []string{t.name + "_min", t.name + "_max", t.name + "_avg", t.name + "_dev"}
 }
 
-// String shows one measurement
-func (m *Measurement) String() string {
-	return fmt.Sprintf("{Measurement %s: wall = %v, system = %v, user = %v}", m.Name, m.Wall, m.User, m.System)
+// String returns the min, max, avg and dev of a Value
+func (t *Value) Values() []string {
+	return []string{fmt.Sprintf("%f", t.store)}
```
Comment (Liamsi) [inline lib/monitor/stats.go]: I tried merging with current development, and stumbled upon this line:
Are you sure this is correct? `%f` assumes a floating value and `t.store` is an array of `float64`

**Comment by Liamsi** [comment]
I fixed the Values output as we discussed. Now the test (`TestStatsString`) is failing. Please fix it first.

**Comment by Liamsi** [comment]
Here is the relevant part of the travis output:
```
=== RUN   TestStatsString
3 : (                   monitor.NewDataFilter: 182) - Filtering: map[]
3 : (                     monitor.ConnectSink:  72) - Connecting to: localhost:10000
2 : (               monitor.(*Monitor).Listen:  86) - Monitor listening for stats on 0.0.0.0 : 10000
3 : (         monitor.(*Monitor).Listen.func1: 103) - Monitor: new connection from [::1]:36766
3 : (                     monitor.ConnectSink:  77) - Connected to sink: localhost:10000
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:36766: &{Name:test_wall Value:0.100214823}
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:36766: &{Name:test_system Value:0}
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:36766: &{Name:test_user Value:0}
--- FAIL: TestStatsString (0.21s)
	stats_test.go:203: The measurement shouldn't contain 0.000000: {Stats: hosts = 10servers = 10[0.000000 0.000000 0.000000 0.000000 0.000000][0.000000 0.000000 0.000000 0.000000 0.000000][0.000000 0.000000 0.000000 0.000000 0.000000]}
FAIL
exit status 1
```

**Comment by Liamsi** [comment]
Before this change: https://github.com/dedis/cothority/pull/263/commits/e1d99b25a9af4eeaa51bcecf048153b8e58e7de1#diff-01f29e29a62f33a20f75389871626f9eL414
the output was: 
```
=== RUN   TestStatsString
3 : (                   monitor.NewDataFilter: 182) - Filtering: map[]
3 : (                     monitor.ConnectSink:  72) - Connecting to: localhost:10000
2 : (               monitor.(*Monitor).Listen:  86) - Monitor listening for stats on 0.0.0.0 : 10000
3 : (         monitor.(*Monitor).Listen.func1: 103) - Monitor: new connection from [::1]:41028
3 : (                     monitor.ConnectSink:  77) - Connected to sink: localhost:10000
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:41028: &{Name:test_wall Value:0.100180381}
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:41028: &{Name:test_system Value:3.200000000000078e-05}
3 : (     monitor.(*Monitor).handleConnection: 175) - Monitor: received a Measure from [::1]:41028: &{Name:test_user Value:8.500000000000174e-05}
2 : (                 monitor.(*Monitor).Stop: 140) - Monitor Stop
--- PASS: TestStatsString (0.20s)
```

**Comment by ineiti** [comment]
Yep, playing around with the `Values`-function - Now I added the `Collect()` before converting in the `String()`-function, and it works out again.

**Comment by Liamsi** [comment]
:+1:

**Pull request merged**